docs/notes/2.23.x.md

```diff
@@ -18,6 +18,10 @@ We offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/spo
 
 ### Backends
 
+#### BUILD
+
+Support for parametrizing grouped parametrizations. (e.g. `**parametrize(resolve=parametrize("a", "b"), ...)`). This works for `___defaults__` as well, as long as it is specified per target type rather than using `all`.
```

Comment:
The `test_default_parametrized_groups_with_parametrizations` seems to use `all`, not per-target-type, so is that now supported?

```suggestion
Support for parametrizing grouped parametrizations. (e.g. `**parametrize(resolve=parametrize("a", "b"), ...)`). This works for `___defaults__` as well.
```

Reply:
Ah, no, should change the test, as it only works because the test runs with a limited number of target types.